packages/opencv-python/meta.yaml

```diff
@@ -26,6 +26,10 @@ requirements:
     - libwebp
     - libtiff
     - setuptools
+  constraint:
+    # ImportError: The 'wheel.bdist_wheel' module has been removed.
+    # Please update your setuptools to v70.1 or later.
+    - wheel < 0.46.1
```

Comment:
This is easy to patch, as we only need to replace all instances of `wheel.bdist_wheel` in `opencv-python`'s sources to `setuptools.command.bdist_wheel.bdist_wheel`.

Please feel free to leave it for this PR if you'd like!

Reply:
That's true, but I would like to avoid patching the source code of the package as much as possible. I think updating the opencv-python version would work, but that requires more work.